payment/channel_factory.py

```diff
@@ -52,17 +43,10 @@ def get_next_channel_id():
 @contextlib.contextmanager
 def get_channel(root_wallet: Blockchain):
     """gets next channel_id from redis, generates address/ tops up and inits sdk."""
-    # TODO: do we need channels for each service? or do we pay the tx fees?
     with get_next_channel_id() as channel_id:
         keys = generate_key(root_wallet, channel_id)
         public_address = keys.address().decode()
-        try:
-            root_wallet.create_wallet(public_address, MEMO_INIT, INITIAL_XLM_AMOUNT)
+        if not root_wallet.read_sdk.does_account_exists(public_address):
```

Comment:
wouldn't be always be created with the new SDK?

Reply:
Im not using the new channel implementation, im still using the redis managed channels